Day5/src/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom/client";
+
+import CardComponent from "./CardComponent";
+import team from "./UserDetails.json";
+import { title } from "./constants";
+import { image } from "./constants";
+import Searchbar from "./SearchBar";
+
+const HeaderComponent = () => {
+  return (
+    <div className="header-container">
+      <h2 className="header-container-left">{title}</h2>
+      <img src={image} alt="banner" />
+    </div>
+  );
+};
+// const data = [
+//   {
+//     img: "https://avatars.githubusercontent.com/iamkirti",
+//     name: "Kirti Sahai",
+//     city: "Bangalore",
+//     company: "Kongsberg Digital",
+//     position: "Development Engineer",
+//     id: "1rfrfr",
+//   },
+// ];
+
+const CardContainer = ({ filteredEmp }) => {
+  return filteredEmp.map((empdetail) => {
```

Comment:
returns can be removed when using arrow functions, if not having multiple statements to execute in the function body.